packages/components/link/src/link.js

```diff
@@ -8,22 +8,49 @@ import { FormattedMessage } from 'react-intl';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 import { filterInvalidAttributes } from '@commercetools-uikit/utils';
 
+export const tones = {
+  primary: 'primary',
+  inverted: 'inverted',
+};
+const linkStates = {
+  normal: 'normal',
+  hover: 'hover',
+};
```

Comment:
This won't really work in TS (once we migrate this component to it) as there is no need for map/constants.
I would keep it simple and use string values.